packages/collector/test/tracing/frameworks/node-fetch/app.mjs

```diff
@@ -0,0 +1,51 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+/* eslint-disable no-console */
+
+'use strict';
+
+// NOTE: c8 bug https://github.com/bcoe/c8/issues/166
+process.on('SIGTERM', () => {
+  process.disconnect();
+  process.exit(0);
+});
+
+import express from 'express';
+import morgan from 'morgan';
+import bodyParser from 'body-parser';
+import getAppPort from '../../../test_util/app-port.js';
+const port = getAppPort();
+import fetch from 'node-fetch';
```

Comment:
I have add a testcase for node-fetch, we only have native fetch testcase, which under protocol folder.